src/compas_fd/fd/fd_numerical_data.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any
+from typing import Tuple
+from typing import List
+from typing import Union
+from typing import Sequence
+from typing import Optional
+from typing_extensions import Annotated
+from nptyping import NDArray
+
+from dataclasses import dataclass
+
+from numpy import asarray
+from numpy import float64
+from numpy import zeros_like
+from scipy.sparse import diags
+
+from compas.numerical import connectivity_matrix
+
+from .result import Result
+
+
+@dataclass
+class FDNumericalData:
```

Comment:
i think this thing is a very good initiative, but it is meant to improve readability not make things more cryptic. why not write out (some of) the names of the attributes?